communication/marketing-team/role-handbooks/storytellers.md

```diff
@@ -9,12 +9,48 @@ TODO add more here
 
 ## Workflow ⚡️
 
-TODO  
-document a short workflow here
-1 should be for those who are comfortable going back and forth on a PR
-another option should include agreeing on a *final* version in a separate
-collaborative doc tool and the sig-docs-liaison or Editor will start the PR
-process
+### Upstream Marketing Process
+
+#### Idea Paths
+
+There are multiple ways to successfully a Kubernetes blog. Here are some of the common paths to help you do so:
+
+
+**To start:** 
+
+* Open a [Contributor Comms Requests issue](https://github.com/kubernetes/community/issues) to share your blog idea, or attend an [Upstream Marketing team](https://github.com/kubernetes/community/tree/master/communication/marketing-team) meeting to discuss it ahead of time
+
+**To write:**
+
+* Create an initial draft -- it is always easier to work from some written down ideas than to do so from scratch
+  * A draft can be written in:
+    * Google Docs or
+    * Markdown (and shared via [Gist](https://gist.github.com/) or [HackMD](https://hackmd.io/))
+* Connect with a storyteller -- if you reach out to Upstream Marketing, you will be paired with someone to help you through writing, editing, and getting your article published
+  * Discussion happens via GitHub issue or through working group meetings
+* This includes working with Upstream Marketing team to polish, and format your blog post
+  * There is not a single canonical format for documents, but we do have [guidelines for effective articles](https://github.com/kubernetes/community/blob/master/communication/marketing-team/blog-guidelines.md#how-to-write-an-effective-blog)
+  * There are further technical considerations by the [Blog team](https://github.com/kubernetes/community/blob/4026287dc3a2d16762353b62ca2fe4b80682960a/sig-docs/blog-subproject/README.md#submit-a-post)
+
+When it's ready, submit the blog post: 
+
+* Go to the `kubernetes/website` repository
+* Open a PR with your added blog post
+  * If you need help with this process, Upstream Marketing will look to pair you with someone for PR and shepherd it through the review cycle
+
+Or if you already have everything you think you need you can:
+
+* Go to the `kubernetes/website` repo and send a PR
+* There's a Kubernetes blog team that will walk you through [the process](https://kubernetes.io/docs/contribute/new-content/blogs-case-studies/)
+
+### Tips
+
+Converting gdoc to markdown?
+
+* go to: https://discuss.kubernetes.io/
+* Open a new topic
+* Paste gdoc into window
+* Profit
```

Comment:
```suggestion
* Now you can copy the markdown text, with all its original formatting, and paste it into your blog post file (this will often be a text file you are editing on your computer from a text editor like VS Code)
* Be sure to add the syntax required at the top of the blog post (see links above)
* Now you are ready to submit it as a PR
```